lib/pbench/test/unit/server/query_apis/test_datasets_update.py

```diff
@@ -228,10 +228,134 @@ def fake_bulk(
         monkeypatch.setattr("elasticsearch.helpers.streaming_bulk", fake_bulk)
 
         response = client.post(
-            f"{server_config.rest_uri}/datasets/publish/random_md5_string1",
+            f"{server_config.rest_uri}/datasets/random_md5_string1",
             headers={"authorization": f"Bearer {pbench_token}"},
             query_string=self.PAYLOAD,
         )
 
         # Verify the failure
         capinternal("Unexpected backend error", response)
+
+    @pytest.mark.parametrize(
+        "owner,access",
+        [
+            ("drb", "public"),
+            ("drb", "private"),
+            ("test", "public"),
+            ("test", "private"),
+        ],
```

Comment:
This is great, but it's not quite 100% coverage. We should be testing cases where only one of the two is specified, whereas you're always specifying both. You should be able to specify `None` in either position, and test what happens when *only* the other query parameter is passed. Then you can just add a couple more test cases here with one or the other set to `None`. (You've already covered the case of specifying neither, in `test_no_query_params`.)

Reply:
Dave, what exactly did you have in mind in terms of handling the `None` arguments?  That is, does Siddardh need to modify line 281 to omit the keys from the query dict when they have `None` values, or is it good/adequate/acceptable to specify the keys with values of `None`?  (And, what about values of `""`?)

Reply:
That was easier than I thought. Cool.